.env.example

```
@@ -0,0 +1,6 @@
+# Appwrite configuration
+APPWRITE_ENDPOINT=https://sfo.cloud.appwrite.io/v1
+APPWRITE_PROJECT_ID=6966f58000347a48517f
+APPWRITE_DATABASE_ID=6966f80700332980dc4e
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The APPWRITE_DATABASE_ID key should go before the APPWRITE_ENDPOINT key

(UnorderedKey)